{"uuid": "7d5f601b-6a9b-44d2-9d2c-8704caa4ade9", "children": ["57f7711d-a748-434e-b784-5bdcdae6b2f0"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: cannot parse capability: browserVersion\nfrom invalid argument: cannot be empty\nStacktrace:\n#0 0x55fed8259133 <unknown>\n#1 0x55fed7f8d966 <unknown>\n#2 0x55fed7faaa51 <unknown>\n#3 0x55fed7fb1551 <unknown>\n#4 0x55fed7faa1b3 <unknown>\n#5 0x55fed7ff1025 <unknown>\n#6 0x55fed7ff0a11 <unknown>\n#7 0x55fed7ff2386 <unknown>\n#8 0x55fed7ff218f <unknown>\n#9 0x55fed7fe99a3 <unknown>\n#10 0x55fed7fbe46a <unknown>\n#11 0x55fed7fbf55e <unknown>\n#12 0x55fed8218cae <unknown>\n#13 0x55fed821c8fe <unknown>\n#14 0x55fed8225f20 <unknown>\n#15 0x55fed821d923 <unknown>\n#16 0x55fed81f0c0e <unknown>\n#17 0x55fed8240b08 <unknown>\n#18 0x55fed8240c97 <unknown>\n#19 0x55fed8251113 <unknown>\n#20 0x7f74a3894b43 <unknown>\n\n", "trace": "  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/andrei/Desktop/code/devops-sandbox/conftest.py\", line 26, in driver\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1684920155518, "stop": 1684920157584}], "start": 1684920155518, "stop": 1684920157610}